examples/QuickJournalSync/Models/JournalEntry.cs

```diff
@@ -0,0 +1,35 @@
+﻿using MongoDB.Bson;
+using QuickJournalSync.Services;
+using Realms;
+
+namespace QuickJournalSync.Models
+{
+    public partial class JournalEntry : IRealmObject
+    {
+        [MapTo("_id")]
+        [PrimaryKey]
+        public ObjectId Id { get; private set; } = ObjectId.GenerateNewId();
+
+        [MapTo("userId")]
+        public string UserId { get; set; }
+
+        public string? Title { get; set; }
+
+        public string? Body { get; set; }
+
+        public DateTimeOffset CreatedDate { get; set; }
```

Comment:
Should we make this private and computed automatically?

Reply:
We can compute this in the constructor

Reply:
I meant the setter 😛

Reply:
That we could 😁